pages/common/pueue-shutdown.md

```diff
@@ -0,0 +1,13 @@
+# pueue shutdown
+
+> Remotely shut down the daemon.
+> Only use this subcommand if the daemon isn't started by a service manager.
+> More information: <https://github.com/Nukesor/pueue>.
+
+- Shutdown the daemon without a service manager:
+
+`pueue shutdown`
+
+- Shutdown the daemon with systemd (Linux only):
```

Comment:
Now that all of those 100 syntax suggestions/comments were cleared, I'm seeing this example in a new light, and I think we should remove it as it's not particularly useful IMO and is only restricted to Linux (Linux distros that use systemd, that is (thought that may be all of them))

Reply:
Actually, I disagree with what I above said.  Just because it's only available on certain distros doesn't mean it's a bad example.  If there's space for more examples, why not add ones that will prove useful to at least some users?  So lets keep this example (or re-add it if it was already removed), and resolve this thread, @she3o.